algebird-core/src/main/scala/com/twitter/algebird/AveragedValue.scala

```diff
@@ -17,17 +17,23 @@ limitations under the License.
 package com.twitter.algebird
 
 object AveragedValue {
+  // TODO: Change to Group[AveragedValue] in 0.13.0
```

Comment:
Can we add an issue and a link to the issue. In my experience TODOs in code are generally ignored.